### ISSUE TITLE
3.x feedback

### ISSUE THREAD

**Issue by AaronJan** [AaronJan/postcss-font-grabber#17]
## 3.0.0-alpha.1

Rewrite the whole project, changed `fontDir` option to `fontDest`, for common usage it's basically the same as `v2.x`.

But inside, there're lots of differences, generally it's significantly simpler (at least it's the goal), and faster (using the new API from PostCSS V8).

Any feature request is welcomed!

```
npm i -D postcss-font-grabber@3.0.0-alpha.2
```

**Comment by FTWinston** [comment]
There seems to be something going wrong with determining the file path to downloaded font files.

I'll put together a proper example and raise a ticket for this, but while it downloads (e.g.) ```src\fonts\temp\96bf74c572b63d425be443075aa896b6.woff2```, I then get an error where the file path is wrong in the generated css:

```ModuleNotFoundError: Module not found: Error: Can't resolve './..☼ontstempﵯ�c572b63d425be443075aa896b6.woff2' in 'C:\SomeProject\src\style'```

The same characters show up in the vscode terminal and in powershell. I've had a quick look through the code and haven't spotted an obvious cause.

**Comment by AaronJan** [comment]
@FTWinston `postcss-font-grabber@3.0.0-alpha.3` published, add a webpack test, fixed a mkdir issue.

**Comment by FTWinston** [comment]
Ah, you beat me to it. Nice one!

Your webpack test fails for me (on Windows, yeah).

If I log ```stats!.toJson().errors```, I see the following:

```
Error: Can't resolve '....outputs\rist\fss\u000fontﲒ�65750f46376b999944dbdeb128.woff2'
Error: Can't resolve '....outputs\rist\fss\u000fontｻ�cb8df965bcfb13204866cef1dc.woff2'
```

It's really weird. In those paths, "dist" has becomes "rist" and "css" has become "fss", but apart from that (and a missing path separator at the start), all the errors are at the start of the file name.

**Comment by FTWinston** [comment]
In ```getNewDeclarationValue``` just before this error occurs, I can see the input value is:

```
url(https://fonts.gstatic.com/s/notosanstc/v11/-nFlOG829Oofr2wohFbTp9i9WyEKIfVZ15Ls5XOFrksA2xrfz7uqxBF_ije-Lcrp9hfGsGeZ-W5oyw.0.woff2)
          format('woff2')
```
and the returned value is 

```
url(..\..\outputs\dist\css\font\b2e443c6bc8eb92ed319c5f3ea877643.woff2)
          format('woff2')
```

And that looks correct to me.

**Comment by AaronJan** [comment]
@FTWinston Added Windows, macOS to the CI testing environments:

<img width="295" alt="image" src="https://user-images.githubusercontent.com/4630940/106834262-0ea72d80-66d0-11eb-8f40-d6fd1fb49cdd.png">

😎

**Comment by AaronJan** [comment]
@FTWinston Considering current status, I published `v3.0.0` 🎉. Couldn't done this without your efforts!

I'll keep adding tests, let me know if you encounter other problem, cheers.

**Comment by FTWinston** [comment]
Excellent, thanks for this. This now does everything I'd hoped it would. Awesome!
